animations/index.md

```diff
@@ -1,159 +1,161 @@
+   <a href="" onMouseOver="document.MyImage.src='/animations/images/animated_crossfade1.gif';" onMouseOut="document.MyImage.src='/animations/images/animated_crossfade1.png';">
+   <img src="/animations/images/animated_crossfade1.png" name="MyImage" height="180" width="160" name="MyImage">
+   </a></td>
+     <td style="width:15%">
+    The <code>AnimatedCrossFade</code> and <code>AnimatedOpacity</code> widgets are two animation examples that use implicit animation widgets.
+     </td>
+     <td style="width:5%">
+     <a href="" onMouseOver="document.MyImage1.src='/animations/images/animated_opacity.gif';" onMouseOut="document.MyImage1.src='/animations/images/animated_opacity.png';">
+     <img src="/animations/images/animated_opacity.png" name="MyImage1" height="180" width="160">
+     </a></td>
+   </tr>
+ </table>
+
+
+
+ <br>
+ For more information including a list of implicit widgets, see [Flutter Implicit Animations](/animations/implicit_widgets).
+
+
+### Transition animation widgets
+
+*Transition animations* are easy to use like implicit animations because transition animation widgets are pre-defined for you, but they also allow you to customize some elements that are included in the explicit animations. With transition animation widgets, you can specify a visual effect that determines how the animation target object is animated. You include the `AnimationController` to control the animation. For example, you can control how the object is made visible or hidden or how to get from the start value to the end value.  
+
+ <table cellpadding="5" style="width:650px">
+  <tr>
+   <td style="width:5%">
+  <a href="" onMouseOver="document.MyImage3.src='/animations/images/rotation_transition.gif';" onMouseOut="document.MyImage3.src='/animations/images/rotation_transition.png';">
+  <img src="/animations/images/rotation_transition.png" name="MyImage3" height="180" width="160" name="MyImage3">
+  </a></td>
+    <td style="width:15%">
+   The <code>RotationTransition</code> and <code>SlideTransition</code> widgets are two animation examples that use transition animation widgets.
+    </td>
+    <td style="width:5%">
+    <a href="" onMouseOver="document.MyImage4.src='/animations/images/slide_transition.gif';" onMouseOut="document.MyImage4.src='/animations/images/slide_transition.png';">
+    <img src="/animations/images/slide_transition.png" name="MyImage4" height="180" width="160">
+    </a></td>
+  </tr>
+</table>
+
+
+
+<br>
+For more information including a list of transition widgets, [Flutter Transition Animations](/animations/transition_widgets).  
+
+
+### Explicit animation widgets
+
+ *Explicit animations* involve building customized animations using the `AnimationBuilder` widget that allows you to customize many of the animation elements that are pre-defined in the implicit and transition animation widgets. When you build explicit animations, you manually add listeners, tickers, and other elements for customization.  
```

Comment:
> Explicit animations involve building customized animations using the `AnimationBuilder` widget that allows you to customize many of the animation elements that are pre-defined in the implicit and transition animation widgets.

AnimatedBuilder (not AnimationBuilder) is one way to build an explicit animation. 

Not sure what "allows you to customize many of the animation elements that are pre-defined in the implicit and transition animation widgets" refers to.  AnimatedBuilder simply uses callback to  build an arbitrary tree of widgets each time its controlling animation ticks.

> When you build explicit animations, you manually add listeners, tickers, and other elements for customization.  

There's a point you could make about potentially having to manage the lifetime of an AnimationController or its listeners however none of that probably bears covering here.

Reply:
Maybe we need to describe the difference between AnimatedBuilder and Animated Controller?

for the 2nd point - we should any details to the explicit animations page.